client/pkg/omni/resources/siderolink/connection_params_test.go

```diff
@@ -5,10 +5,14 @@
 package siderolink_test
 
 import (
+	"bytes"
+	"errors"
+	"io"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
```

Comment:
should be `.v3` (we shouldn't use .v2 anywhere)

Reply:
oh, yeah, that's automatic import, will fix